_episodes/11-hpc-intro.md

```diff
@@ -56,7 +56,7 @@ Using a cluster often has the following advantages for researchers:
   projects.
 * **Efficiency.** Many HPC systems operate a pool of resources that are drawn on by many users. In
   most cases when the pool is large and diverse enough the resources on the system are used almost
-  constantly.
+  constantly, with individual users only using (and paying for) a fraction of the system.
```

Comment:
I'd indicate that they are "possibly" paying for it

Reply:
No objection.